Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift

```diff
@@ -31,8 +31,20 @@ struct PlaceDataSourceError: Error {
 
 class PlaceCarouselViewController: UIViewController {
 
+    fileprivate let minimumTimeAtLocationBeforeFetchingEvents: TimeInterval = 15 * 60
+    fileprivate let currentLocationIdentifier = "CURRENT_LOCATION"
     fileprivate let MIN_SECS_BETWEEN_LOCATION_UPDATES: TimeInterval = 1
-    fileprivate var timeOfLastLocationUpdate: Date?
+
+    private let timeOfLastLocationUpdateKey = "timeOfLastLocationUpdate"
+
+    fileprivate var timeOfLastLocationUpdate: Date? {
+        get {
+            return UserDefaults.standard.value(forKey: timeOfLastLocationUpdateKey) as? Date
```

Comment:
yes, that is the point. This allows our check for `timeOfLastLocationUpdate` in `locationManager(_:didUpdateLocations:)` to still work, so the very first time we start the app, when we have never fetched a location before, we can process the new location and set a value here for the first time.

`UserDefaults.standard.value(forKey:)` is there because `UserDefaults` is backed by a hashtable and inherits that function through there. I'll update to use `UserDefaults.standard.object(forKey:)` to make it use the documented part of the api (the old apple docs used to give you links to the inheritance tree, but I can't see them in the updated docs)